src/wallet/wallet2.h

```diff
@@ -1201,7 +1203,18 @@ namespace tools
       a & m_scanned_pool_txs[1];
       if (ver < 20)
         return;
-      a & m_subaddresses.parent();
+      if (ver < 31)
+      {
+        using pair_type = std::pair<crypto::public_key, cryptonote::subaddress_index>;
+        std::unordered_map<crypto::public_key, cryptonote::subaddress_index> subaddresses;
```

Comment:
I noticed you type-aliasing, but then not using the type alias on the immadiate next line. Why is that?